azurelinuxagent/common/utils/textutil.py

```diff
@@ -396,3 +398,20 @@ def format_memory_value(unit, value):
         raise TypeError('Value must be convertible to a float')
 
     return int(value * units[unit])
+
+def str_to_encoded_ustr(s, encoding='utf-8'):
```

Comment:
not sure i understand what the function is trying to do. could you add a doctring? thanks

Reply:
Make sense, I'll add that.

Reply:
Done

Reply:
thanks